src/main/java/edu/cornell/kfs/paymentworks/util/PaymentWorksUtil.java

```diff
@@ -162,25 +165,33 @@ protected String trimFieldToMax(String field, String fieldName) {
 		return returnField;
 	}
 
-	/*
-	 * KPS-316 format telephone number to xxx-yyy-zzzz
-	 */
-	protected String convertPhoneNumber(String phoneNbr) {
-		// first clear out any non-numeric characters
-		String phone = phoneNbr.replaceAll("[^0-9]", "");
-		StringBuilder p = new StringBuilder(phone.substring(0, 3)).append("-");
-		p.append(phone.substring(3, 6)).append("-");
-		p.append(phone.substring(6, phone.length()));
-		phone = p.toString();
-		return phone;
+	public String convertPhoneNumber(String phoneNumber) {
+		return getPhoneNumberService().formatNumberIfPossible(phoneNumber);
 	}
 
-	protected DataDictionaryService getDataDictionaryService() {
+	private String removeNonNumberCharacters(String phoneNbr) {
```

Comment:
Is this method still being used? If not, I would remove it. If so, I would spell out phoneNumber here as well.